src/media/js/previews-buttons.js

```diff
@@ -21,8 +21,8 @@ define('previews-buttons',
 
         $container.find('.arrow-btn-prev, .arrow-btn-next').remove();
 
-        var $prevBtn = $('<a class="arrow-btn arrow-btn-prev"></a>');
-        var $nextBtn = $('<a class="arrow-btn arrow-btn-next"></a>');
+        var $prevBtn = $('<a class="arrow-btn arrow-btn-prev" data-no-transparent="true"></a>');
+        var $nextBtn = $('<a class="arrow-btn arrow-btn-next" data-no-transparent="true"></a>');
```

Comment:
I don't think we need `="true"`, just the presence of the attribute name should be enough, e.g.

https://github.com/mozilla/fireplace/blob/ac3ea0654011c1fca4387c9fcb2693076f1d525d/src/templates/games/index.html#L40